commands/wallet_delete.go

```diff
@@ -0,0 +1,57 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/mesg-foundation/core/utils/pretty"
+	"github.com/spf13/cobra"
+	survey "gopkg.in/AlecAivazis/survey.v1"
+)
+
+type walletDeleteCmd struct {
+	baseCmd
+
+	address      common.Address
+	noPassphrase bool
+	passphrase   string
+
+	e WalletExecutor
+}
+
+func newWalletDeleteCmd(e WalletExecutor) *walletDeleteCmd {
+	c := &walletDeleteCmd{e: e}
+	c.cmd = newCommand(&cobra.Command{
+		Use:     "delete",
+		Short:   "Delete an account",
+		Long:    "Delete an account from the wallet",
+		Example: "mesg-core wallet delete 0x0",
+		Args:    cobra.ExactArgs(1),
+		RunE:    c.runE,
+	})
+
+	c.cmd.Flags().BoolVarP(&c.noPassphrase, "no-passphrase", "n", c.noPassphrase, "Leave passphrase empty")
```

Comment:
why is there a noPassphrase ? we should always have one no ?

Reply:
@NicolasMahe said that is ok to do not have passpharse, so I added the flag to avoid creating wallet without pass by mistake